classes/person.rb

```diff
@@ -11,6 +11,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] Please add an `add_rental`  method to add an association to rentals. I can be something like this 👍 

```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Oh sure thank you for the suggestion. I am thinking to pass the whole rental object as parameter to `add_rental` method. Like this, I will not have to create an instance of `Rental` inside the method again.